deploy.yml

```diff
@@ -95,6 +130,17 @@ spec:
         - "--container-runtime-endpoint=/run/crio/crio.sock"
         # uncomment this if you want to store iptables rules
         - "--pod-iptables=/var/lib/multi-networkpolicy/iptables"
+        # uncomment this if you need to accept link-local address traffic
+        #- "--allow-ipv6-src-prefix=fe80::/10"
+        #- "--allow-ipv6-dst-prefix=fe80::/10"
+        # uncomment this if you need to add custom iptables rules defined above configmap
+        #- "--custom-v4-ingress-rule-file=/etc/multi-networkpolicy/rules/custom-v4-rules.txt"
+        #- "--custom-v4-egress-rule-file=/etc/multi-networkpolicy/rules/custom-v4-rules.txt"
+        #- "--custom-v6-ingress-rule-file=/etc/multi-networkpolicy/rules/custom-v6-rules.txt"
+        #- "--custom-v6-egress-rule-file=/etc/multi-networkpolicy/rules/custom-v6-rules.txt"
```

Comment:
I noticed these rules are commented out here but not in the e2e yaml. In `options.go` you also use define stringVars for these rule files i.e. 
```fs.StringVar(&o.customIPv4IngressRuleFile, "custom-v4-ingress-rule-file", "", "custom rule file for IPv4 ingress")``` .
This rule file is then parsed in `options.go` (via ParseCustomRuleFile) for the actual rule. I am wondering if commenting out the filepath here will cause any problems due to missing filepath definition?

Reply:
This option just injects iptables rules so it is not tested in e2e because e2e mainly tests iptables generated by multi-networkpolicy-iptables. This option just injects the rule user created, so currently we don't test it yet.

In addition, the file `/etc/multi-networkpolicy/rules/custom-v4-rules.txt` in deploy.yml is available by configmap as follows, hence if user just remove comments, then given rule will be available (but I wish user will uncomment it once they understand what they are going to do that, actually).

https://github.com/k8snetworkplumbingwg/multi-networkpolicy-iptables/blob/583368821429080cfb7aaefaa4faa663ba326b12/deploy.yml#L56-L91